packages/link/lib/text-link.module.scss

```diff
@@ -12,19 +12,21 @@ a.textlink {
   $margin-ltr: 0 -#{$gap} 0 0;
   $margin-rtl: 0 0 0 -#{$gap};
   @include mixins.rtl(margin, $margin-ltr, $margin-rtl);
+
   > * {
     $child-margin-ltr: 0 #{$gap} 0 0;
     $child-margin-rtl: 0 0 0 #{$gap};
     @include mixins.rtl(margin, $child-margin-ltr, $child-margin-rtl);
   }
 
   // Remove text-decoration from child icon on hover.
-  & > i:before {
+  & > i::before {
```

Comment:
TIL abt pseudo-classes vs pseudo-elements

Reply:
me too! i always took for granted that browsers would just accept either syntax to avoid breaking everyone's CSS